test/metabase/driver_test.clj

```diff
@@ -183,8 +183,12 @@
                                 "sum"        true}}]
             formatted-query (driver/prettify-native-form :mongo query)]
 
-        (testing "Formatting a non-sql query returns the same query"
-          (is (= query formatted-query)))
+        (testing "Formatting a mongo query returns a JSON-like string"
+          (is (= "[\n  {\n    \"$group\": {\n      \"_id\": {\n        \"created_at\": {\n          \"$let\": {\n            \"vars\": {\n              \"parts\": {\n                \"$dateToParts\": {\n                  \"timezone\": \"UTC\",\n                  \"date\": \"$created_at\"\n                }\n              }\n            },\n            \"in\": {\n              \"$dateFromParts\": {\n                \"timezone\": \"UTC\",\n                \"year\": \"$$parts.year\",\n                \"month\": \"$$parts.month\",\n                \"day\": \"$$parts.day\"\n              }\n            }\n          }\n        }\n      },\n      \"sum\": {\n        \"$sum\": \"$tax\"\n      }\n    }\n  },\n  {\n    \"$sort\": {\n      \"_id\": 1\n    }\n  },\n  {\n    \"$project\": {\n      \"_id\": false,\n      \"created_at\": \"$_id.created_at\",\n      \"sum\": true\n    }\n  }\n]"
```

Comment:
These big strings could be more readable by splitting into multiple strs and use `str` / `str/join`
```
(str "[\n"
     "  {\n" 
     ...
     "]"
```